app/(app)/articles/[slug]/page.tsx

```diff
@@ -119,11 +119,15 @@ const ArticlePage = async ({ params }: Props) => {
 
           {isTiptapContent ? (
             <div
-              dangerouslySetInnerHTML={{ __html: renderedContent }}
+              dangerouslySetInnerHTML={{ __html: renderedContent ?? "" }}
```

Comment:
If we fallback to `""` should we not have an error thrown?

Reply:
So without the extra `|| ""` you get a Typescript error warning you that if it ends up getting passes a null value, there will be problems.

It seems like an easy fix, but if you wanted to default to something else I will be happy to change it 👍🏻

Reply:
I think if there is no content it should probably be notFound since there is no data (or content) for that page? That make sense?

Reply:
Great I will just bring in the `NotFound` component then, and replace the empty string instance with that 👍